File: 01-Front/Saniteau/Models/Compteurs/Compteur.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Saniteau.Models.Compteurs
{
    public class Compteur
    {
        public int IdCompteur { get; set; }
        public string NumeroCompteur { get; set; }
        public bool CompteurEstPose { get; set; }
        public bool compteurEstAppaire { get; set; }
        public PDL PDL { get; set; }
        public int IdAbonne { get; set; }

        public Compteur()
        {

        }

        public Compteur(int idCompteur, string numeroCompteur, bool compteurEstPose, bool compteurAppairé, PDL pdl, int idAbonne)
        {
            IdCompteur = idCompteur;
            NumeroCompteur = numeroCompteur;
            CompteurEstPose = compteurEstPose;
            compteurEstAppaire = compteurAppairé;
            PDL = pdl;
            IdAbonne = idAbonne;
        }
    }
}

File: 01-Front/Saniteau/client-src/model/compteur/Compteur.ts
import { PDL } from "./PDL";

export class Compteur {
    public idCompteur: number;
    public numeroCompteur: string;
    public compteurEstPose: boolean;
    public compteurEstAppaire: boolean;
    public pdl: PDL;
    public idAbonne: number;


    constructor(idCompteur: number, numeroCompteur: string, compteurEstPose: boolean, compteurEstAppaire: boolean, pdl: PDL, idAbonne: number) {
        this.idCompteur = idCompteur;
        this.numeroCompteur = numeroCompteur;
        this.compteurEstPose = compteurEstAppaire;
        this.compteurEstAppaire = compteurEstPose;
        this.pdl = pdl;
        this.idAbonne = idAbonne;
    }

}